source/analysis/AM3DBinningAB.cpp

```diff
@@ -472,8 +472,8 @@ void AM3DBinningAB::setSumRangeMin(int sumRangeMin)
     cacheUpdateRequired_ = true;
     dirtyIndices_.clear();
     reviewState();
-    emitValuesChanged();
```

Comment:
Was the problem here that emitValuesChanged() was called before setModified(true)?